packages/react/src/auto/mui/MUIAutoForm.tsx

```diff
@@ -69,15 +69,18 @@ export const MUIAutoFormComponent = <
     },
   };
 
-  const humanizedOperationName = useMemo(() => humanizeCamelCase(action.operationName), [action.operationName]);
+  const formTitle = useMemo(
```

Comment:
I don't think this needs to be a memo -- I think we can just derive the title directly

Reply:
Updated to be used directly with no memo